data/avro-schema/src/main/resources/avro/net/corda/data/ledger/utxo/contract.verification/VerifyContractsResponse.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "record",
+  "name": "VerifyContractsResponse",
+  "doc": "The response to an {@link VerifyContractsRequest}.",
+  "namespace": "net.corda.ledger.utxo.contract.verification",
+  "fields": [
+    {
+      "name": "result",
+      "type": "boolean",
```

Comment:
This might be a bit strict, though I agree right now we can only envisage a boolean response. Perhaps this should be an Enum with VERIFIED, INVALID for now and also add a message field to let us put some logging into flow worker.

Does this also need the TxId?

Reply:
Replaced with enum and added message filed.

Flow side is sending transaction as an argument to verification, so it has TxId